### PR TITLE
Add the readonly Content Type field (innholdstype) to Dokumentversjon

### PR DESCRIPTION
**Pull request by dkumanov**

### CODE REVIEW

File: src/main/java/com/documaster/rms/noark/ws/noarkentities/Dokumentversjon.java [hunk 1]
```diff
@@ -29,6 +29,9 @@ public class Dokumentversjon extends NoarkEntityBase<Dokumentversjon> {
 	@JsonIgnore
 	private String filnavn;
 
+	@JsonIgnore
```
Comment (ivaylomitrev) [inline src/main/java/com/documaster/rms/noark/ws/noarkentities/Dokumentversjon.java:32]: I believe the field should be writeable too.

**Review by ivaylomitrev — CHANGES_REQUESTED** [review]

File: src/main/java/com/documaster/rms/noark/ws/noarkentities/Dokumentversjon.java (hunk 1 above)
Comment (dkumanov) [reply src/main/java/com/documaster/rms/noark/ws/noarkentities/Dokumentversjon.java:32]: The field is not writable, it's system managed as described in the documentation PR. I doublechecked by trying to set the field and the request failed for that reason.

Comment (ivaylomitrev) [reply src/main/java/com/documaster/rms/noark/ws/noarkentities/Dokumentversjon.java:32]: Ah, you are right! We should probably think about exposing it for writing in the future, but that wouldn't be a part of this PR then.

Thanks for the clarification.

**Review by ivaylomitrev — APPROVED** [review]

**Review by orlinDocumaster — COMMENTED** [review]
no comments

**Pull request merged**